testing/sdk_install.py

```diff
@@ -32,6 +32,24 @@
 _installed_service_names = set([])
 
 
+def get_package_json(package, version=None, options=None):
```

Comment:
Note: When this merges please upstream this to dcos-commons

Reply:
Is this the correct module to put it in?

Reply:
I think it makes sense.

You're basically just getting the rendered marathon app to do a "manual" install